api/v1beta1/hostedcluster_types.go

```diff
@@ -86,7 +86,9 @@ const (
 	// AWSCredentialsFileSecretKey defines the Kubernetes secret key name that contains
 	// the customer AWS credentials in the unmanaged authentication strategy for AWS KMS secret encryption
 	AWSCredentialsFileSecretKey = "credentials"
-
+	// KubeVirtExternalInfraCredentialsSecretKey defines the kubeconfig needed for the hosted control plane operator
+	// to access and manage kubevirt workloads on an external infra cluster.
+	KubeVirtExternalInfraCredentialsSecretKey = "kubevirt-external-credentials"
```

Comment:
is this used yet?

Reply:
eventually I didn't use that const. removed.